ambry-clustermap/src/main/java/com.github.ambry.clustermap/CompositeClusterManager.java

```diff
@@ -289,8 +303,7 @@ private boolean areEqual(List<PartitionId> partitionListOne, List<PartitionId> p
  */
 class DuplicatingInputStream extends FilterInputStream {
   private enum Mode {
-    SAVE_READS,
-    SERVE_DUPLICATES
+    SAVE_READS, SERVE_DUPLICATES
```

Comment:
does this have to be formatted this way?

Reply:
right, this is due to file format.